stories/components/Menu/Menu.stories.tsx

```diff
@@ -90,6 +95,76 @@ const MenuStory: React.FC = () => {
             ]}
           />
         </Container>
+        <Container>
+          <Menu
+            aria-label="Chroma menu options children"
+            anchorElement={
+              <IconButton
+                aria-label="Click to open children only menu"
+                icon={MoreHorizontal}
+              />
+            }
+            items={[
+              <MenuGroupHeading>First Section</MenuGroupHeading>,
```

Comment:
My only thought was this: is there an advantage of not abstracting away `MenuGroupHeading` inside `Menu` via some sort of prop? I thought this because `MenuGroupHeading` is very specific and meant to be used within the `Menu` context. I thought that having the menu take something like the following could work:

```js
sections: [
   { 
       title: 'Header Title', // this would be optional
       items: [],
   }
]
```

Reply:
@aecorredor good question. I went this route because there was prior art in the library -- this is very similar to how `Select` works, for example: https://github.com/lifeomic/chroma-react/blob/master/stories/components/Select/Select.stories.tsx#L136-L148

Reply:
It was also a non-breaking change, which I felt was important. Your proposed solution could also be done in a non-breaking way -- just mentioning for clarity.

Reply:
Makes sense!